io-engine/src/subsys/nvmf/transport.rs

```diff
@@ -57,8 +62,49 @@ pub async fn add_tcp_transport() -> Result<(), Error> {
     };
 
     let _result = r.await.unwrap();
-
     debug!("Added TCP nvmf transport");
+
+    if add_rdma {
+        info!("Adding RDMA transport for Mayastor Nvmf target");
+        let mut opts = cfg.nvmf_tgt_conf.opts_rdma.into();
+        let transport = unsafe {
+            spdk_nvmf_transport_create(RDMA_TRANSPORT.as_ptr(), &mut opts)
+        };
+
+        let ret = transport
+            .to_result(|_| Error::Transport {
```

Comment:
not sure what's happening here, we're adding error but then we seem to just clear the error below?